2022-01-25 21:53:41,528 INFO    MainThread:21612 [wandb_setup.py:_flush():71] setting env: {}
2022-01-25 21:53:41,528 INFO    MainThread:21612 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-25 21:53:41,528 INFO    MainThread:21612 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d02_train\wandb\run-20220125_215341-2qt4loa1\logs\debug.log
2022-01-25 21:53:41,528 INFO    MainThread:21612 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d02_train\wandb\run-20220125_215341-2qt4loa1\logs\debug-internal.log
2022-01-25 21:53:41,528 INFO    MainThread:21612 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001B96AB2B1C0>
2022-01-25 21:53:41,529 INFO    MainThread:21612 [wandb_init.py:init():404] calling init triggers
2022-01-25 21:53:41,529 INFO    MainThread:21612 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-25 21:53:41,529 INFO    MainThread:21612 [wandb_init.py:init():449] starting backend
2022-01-25 21:53:41,529 INFO    MainThread:21612 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2022-01-25 21:53:41,532 INFO    MainThread:21612 [backend.py:ensure_launched():199] starting backend process...
2022-01-25 21:53:41,843 INFO    MainThread:21612 [backend.py:ensure_launched():204] started backend process with pid: 30984
2022-01-25 21:53:41,844 INFO    MainThread:21612 [wandb_init.py:init():458] backend started and connected
2022-01-25 21:53:41,846 INFO    MainThread:21612 [wandb_run.py:_label_probe_notebook():917] probe notebook
2022-01-25 21:53:41,846 INFO    MainThread:21612 [wandb_run.py:_label_probe_notebook():927] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-01-25 21:53:41,846 INFO    MainThread:21612 [wandb_init.py:init():519] updated telemetry
2022-01-25 21:53:41,849 INFO    MainThread:21612 [wandb_init.py:init():549] communicating current version
2022-01-25 21:53:43,259 INFO    MainThread:21612 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-25 21:53:43,259 INFO    MainThread:21612 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-01-25 21:53:43,502 INFO    MainThread:21612 [wandb_init.py:init():592] starting run threads in backend
2022-01-25 21:53:44,419 INFO    MainThread:21612 [wandb_run.py:_console_start():1816] atexit reg
2022-01-25 21:53:44,420 INFO    MainThread:21612 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2022-01-25 21:53:44,420 INFO    MainThread:21612 [wandb_run.py:_redirect():1727] Wrapping output streams.
2022-01-25 21:53:44,421 INFO    MainThread:21612 [wandb_run.py:_redirect():1751] Redirects installed.
2022-01-25 21:53:44,422 INFO    MainThread:21612 [wandb_init.py:init():619] run started, returning control to user process
2022-01-25 21:53:45,422 INFO    MainThread:21612 [wandb_init.py:_pause_backend():284] pausing backend
2022-01-25 21:53:45,422 INFO    MainThread:21612 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-25 21:57:30,466 INFO    MainThread:21612 [wandb_init.py:_resume_backend():293] resuming backend
2022-01-25 21:57:30,467 INFO    MainThread:21612 [wandb_init.py:_pause_backend():284] pausing backend
2022-01-25 21:57:30,468 INFO    MainThread:21612 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-25 21:57:37,028 INFO    MainThread:21612 [wandb_init.py:_resume_backend():293] resuming backend
2022-01-25 21:57:37,030 INFO    MainThread:21612 [wandb_init.py:_pause_backend():284] pausing backend
2022-01-25 21:57:37,030 INFO    MainThread:21612 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-25 21:57:46,467 INFO    MainThread:21612 [wandb_init.py:_resume_backend():293] resuming backend
2022-01-25 21:57:46,469 INFO    MainThread:21612 [wandb_init.py:_pause_backend():284] pausing backend
2022-01-25 21:57:46,469 INFO    MainThread:21612 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-25 21:58:23,529 INFO    MainThread:21612 [wandb_init.py:_resume_backend():293] resuming backend
2022-01-25 21:58:23,538 INFO    MainThread:21612 [wandb_setup.py:_flush():71] setting env: {}
2022-01-25 21:58:23,539 INFO    MainThread:21612 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-25 21:58:23,539 INFO    MainThread:21612 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d02_train\wandb\run-20220125_215823-1tft16ig\logs\debug.log
2022-01-25 21:58:23,540 INFO    MainThread:21612 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d02_train\wandb\run-20220125_215823-1tft16ig\logs\debug-internal.log
2022-01-25 21:58:23,540 INFO    MainThread:21612 [wandb_init.py:init():404] calling init triggers
2022-01-25 21:58:23,540 INFO    MainThread:21612 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-25 21:58:23,540 INFO    MainThread:21612 [wandb_init.py:init():424] re-initializing run, found existing run on stack: 2qt4loa1
2022-01-25 21:58:23,540 INFO    MainThread:21612 [wandb_run.py:finish():1463] finishing run austinbergstrom/uncategorized/2qt4loa1
2022-01-25 21:58:23,540 INFO    MainThread:21612 [jupyter.py:save_history():429] not saving jupyter history
2022-01-25 21:58:23,541 INFO    MainThread:21612 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-25 21:58:23,541 INFO    MainThread:21612 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-01-25 21:58:23,541 INFO    MainThread:21612 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 0
2022-01-25 21:58:23,541 INFO    MainThread:21612 [wandb_run.py:_restore():1758] restore
2022-01-25 21:58:24,253 INFO    MainThread:21612 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 650
  total_bytes: 650
}

2022-01-25 21:58:24,428 INFO    MainThread:21612 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 650
  total_bytes: 650
}

2022-01-25 21:58:24,661 INFO    MainThread:21612 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 650
  total_bytes: 650
}

2022-01-25 21:58:24,795 INFO    MainThread:21612 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 650
  total_bytes: 5030
}

2022-01-25 21:58:24,896 INFO    MainThread:21612 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 650
  total_bytes: 5056
}

2022-01-25 21:58:25,015 INFO    MainThread:21612 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 650
  total_bytes: 5056
}

2022-01-25 21:58:25,119 INFO    MainThread:21612 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 650
  total_bytes: 5056
}

2022-01-25 21:58:25,221 INFO    MainThread:21612 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5056
  total_bytes: 5056
}

2022-01-25 21:58:25,323 INFO    MainThread:21612 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5056
  total_bytes: 5056
}

2022-01-25 21:58:25,428 INFO    MainThread:21612 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5056
  total_bytes: 5056
}

2022-01-25 21:58:25,530 INFO    MainThread:21612 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5056
  total_bytes: 5056
}

2022-01-25 21:58:25,839 INFO    MainThread:21612 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5056
  total_bytes: 5056
}

2022-01-25 21:58:26,291 INFO    MainThread:21612 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5056
  total_bytes: 5056
}
local_info {
}

2022-01-25 21:58:27,423 INFO    MainThread:21612 [wandb_run.py:_append_files():2184] logging synced files
